tool/gravity/cli/utils.go

```diff
@@ -238,13 +238,19 @@ func isCancelledError(err error) bool {
 
 func watchReconnects(ctx context.Context, cancel context.CancelFunc, watchCh <-chan rpcserver.WatchEvent) {
 	go func() {
-		for event := range watchCh {
-			if event.Error == nil {
-				continue
+		for {
+			select {
+			case event := <-watchCh:
```

Comment:
For robustness I'd handle a closed channel -- it looks like if this were to happen it'd result in a tight spin when event.Error defaults to its zero value of nil.

Reply:
Done.